### PR TITLE
Edit profile

### PR DESCRIPTION
**Pull request by grishanych**

### CODE REVIEW

File: src/Components/edit_profile/edit_profile.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Component } from 'react';
+import { Modal} from 'antd';
+import 'antd/dist/antd.css';
+import './edit_profile.scss'
+import FormEditProfile from "./form_edit_profile";
+
+class EditProfile extends Component {
```
Comment (lhalam) [inline src/Components/edit_profile/edit_profile.js]: change name to Profile

File: src/Components/edit_profile/edit_profile.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    state = {
+        showModal: false
```
Comment (lhalam) [inline src/Components/edit_profile/edit_profile.js]: showEditProfileModeal

File: src/Components/edit_profile/edit_profile.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
+    handleOk = () => {
```
Comment (lhalam) [inline src/Components/edit_profile/edit_profile.js]: handleShowEditProfileModeal

File: src/Components/edit_profile/edit_profile.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                    width={880}
+                    className='registration_modal'
+                    visible={this.state.showModal}
+                    onCancel={() => { this.setState({ showModal: !this.state.showModal }) }}
+                    footer={null} >
+                       <FormEditProfile />
+
+                    
+
+                </Modal>
```
Comment (lhalam) [inline src/Components/edit_profile/edit_profile.js]: move to ModalEditProfile

File: src/Components/edit_profile/form_edit_profile.js [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+import 'antd/dist/antd.css';
+import './edit_profile.scss'
+
+class FormEditProfile extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            showPasswordInput: false
+        }
+    }
```
Comment (lhalam) [inline src/Components/edit_profile/form_edit_profile.js]: state = {
            chagetPasword: false
        }

File: src/Components/edit_profile/form_edit_profile.js [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+
+    toggleCheckboxPassword = () => {
+        if (Checkbox.checked == true) {
+            this.setState({ showPasswordInput: true })
+        }
+        else { this.state({ showPasswordInput: false }) }
+    }
```
Comment (lhalam) [inline src/Components/edit_profile/form_edit_profile.js]: this.setState({ chagetPasword: !this.state.chagetPasword});

File: src/Components/edit_profile/form_edit_profile.js [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+                <Form.Item
+                    name="photo"
+                    className="edit-input"
+                > <span className='change-photo'>Фото&nbsp;< QuestionCircleOutlined />:</span>
+                    <span className='upload-photo'>< UploadOutlined />&nbsp; Завантажити фото</span>
+                </Form.Item>
+
+                <Checkbox.Group>
+                    <Col className='row-style'>
+                        <Checkbox onClick={() => { this.setState({ showPasswordInput: true }) }}><span className='advancedSearchSpan'>Змінити пароль</span></Checkbox>
```
Comment (lhalam) [inline src/Components/edit_profile/form_edit_profile.js]: onClick={this.handleChagetPasword} checked={this.state.chagetPasword}

File: src/Components/edit_profile/form_edit_profile.js [hunk 8]
```diff
@@ -0,0 +1,230 @@
...
+                    </Col>
+                </Checkbox.Group>
+                {this.state.showPasswordInput ?
```
Comment (lhalam) [inline src/Components/edit_profile/form_edit_profile.js]: {this.state.showPasswordInput &&  <div></div>}

File: src/Components/edit_profile/form_edit_profile.js [hunk 9]
```diff
@@ -0,0 +1,230 @@
...
+                            rules={[
+                                {
+                                    required: true,
+                                    message: 'Підтвердіть пароль!',
+                                },
+                            ]}                >
+                            <Input.Password placeholder='Підтвердіть новий пароль' />
+                        </Form.Item>
+                    </div>
+                    : null}
```
Comment (lhalam) [inline src/Components/edit_profile/form_edit_profile.js]: remove this

File: src/Components/edit_profile/modal_edit_profile.js [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+    state = {
+        showEditProfileModal: false
+    }
+    handleShowEditProfileModal = () => {
+        this.setState({ showEditProfileModal: !this.state.showEditProfileModal });
+    };
+    render() {
+        return (
+            <div>
+                <a onClick={() => { this.handleShowEditProfileModal() }} >Редагувати профіль</a>
```
Comment (lhalam) [inline src/Components/edit_profile/modal_edit_profile.js]: onClick={ this.handleShowEditProfileModal}

File: src/Components/edit_profile/modal_edit_profile.js [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+                <Modal
+                    width={880}
+                    className='registration_modal'
+                    visible={this.state.showEditProfileModal}
+                    onCancel={() => { this.setState({ showEditProfileModal: !this.state.showEditProfileModal }) }}
```
Comment (lhalam) [inline src/Components/edit_profile/modal_edit_profile.js]: onCancel={ this.handleShowEditProfileModal}

File: src/Components/edit_profile/modal_edit_profile.js [hunk 12]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Component } from "react/cjs/react.production.min";
```
Comment (lhalam) [inline src/Components/edit_profile/modal_edit_profile.js]: fix

**Review by Sel1nd — APPROVED** [review]

**Review by tytomyr — APPROVED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**